doc/short_tutorial.rst

```diff
@@ -0,0 +1,123 @@
+.. SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _short_tutorial:
```

Comment:
Generally, the approach is good; there are minor formatting/english issues that the pre-commit should prefilter.
Maybe we can scout later together why the pre-commit is not well working

Reply:
Generally, avoid double spacing "  ", and after commas there must be a space. Moreover I saw somewhere "counties" for "countries" and alike. Minor issues anyway :)